### ISSUE TITLE
Get data across all tutorials

### ISSUE THREAD

**Issue by nomadscientist** [galaxyproject/training-material#5610]
Since the invent of the Activity Bar, and a button that says 'Upload data' (which to me, as a user, would not also mean 'retrieve data from other sources', aka what's available in the paste-fetch feature that we use for all training), the tutorials that all start with Get Data (which I also think is a standard format from the PDKN automated skeleton generator) now no longer correspond with the interface.

They should not go to Get Data, which has a TON of tools in it. Nor should they use the Import button which is linked (but doesn't work (https://usegalaxy.eu/training-material/topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.html#galaxy-basics-for-everyone).  They need to go to the Upload data button.

So basically, we need to edit the beginning of a ton of tutorials to match this now.

**Comment by hexylena** [comment]
😭😭😭

**Comment by nomadscientist** [comment]
Christ
